README.md

```diff
@@ -341,7 +341,10 @@ Note that only single-character-prompts are allowed. setting `MCFLY_PROMPT` to `
 
 ### Database Location
 
-McFly stores its SQLite database in the standard location for the OS. On OS X, this is in `~/Library/Application Support/McFly` and on Linux it is in `$XDG_DATA_DIR/mcfly/history.db` (default would be `~/.local/share/mcfly/history.db`). For legacy support, if `~/.mcfly/` exists, it is used instead.
```

Comment:
I think instead of migrating, it should keep using the old location if it exists.